relative-paths.test.js

```diff
@@ -0,0 +1,42 @@
+/* global test, expect, afterEach */
+const execa = require('execa')
+const path = require('path')
+
+const CLI = path.resolve(__dirname, `./dist/src/index.js`)
```

Comment:
@KnisterPeter This is the reason for that additional build step -- I want to be able to reference the compiled version of this file in the test.